### PR TITLE
Add EpsilonThreshold argument for DetectVG

### PR DESCRIPTION
**Pull request by Alanocallaghan**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1499,3 +1499,22 @@ overdispersion parameters
 
 version 2.1.7 (2020-09-01)
 - Bugfix in format method for ResultVG class.
```
Comment (catavallejos) [inline NEWS:1501]: This should be

```
version 2.1.8 (2020-09-23)
- list of changes
```

File: DESCRIPTION [hunk 2]
```diff
@@ -1,8 +1,8 @@
 Package: BASiCS
 Type: Package
 Title: Bayesian Analysis of Single-Cell Sequencing data
-Version: 2.1.7
-Date: 2020-09-01
+Version: 2.2.0
```
Comment (catavallejos) [inline DESCRIPTION:4]: We do not change y in the version number. That's done by the BioC team once the new release is up. 

Version number should be 2.1.8
Date should be 2020-09-23

**Review by catavallejos — CHANGES_REQUESTED** [review]

File: NEWS (hunk 1 above)
Comment (Alanocallaghan) [reply NEWS:1501]: 2.1.6 is the latest on Bioc, I was going by that

Comment (catavallejos) [reply NEWS:1501]: (sorry to be painful about this - trying to be consistent with BioC rules)

We should bump z for *every* commit. Technically, we are now in version 2.1.? :-)

Comment (catavallejos) [reply NEWS:1501]: ... I've lost the count about how many commits!

Comment (Alanocallaghan) [reply NEWS:1501]: In that case it's maybe better to merge and then I'll rebase - shouldn't be hard and is really much easier to track things that way

For future PRs it's probably better to squash and merge

Comment (catavallejos) [reply NEWS:1501]: OK. 

And agreed about future PR.

**Pull request merged**